homeassistant/components/device_tracker/bluetooth_tracker.py

```diff
@@ -99,9 +100,17 @@ def update_bluetooth(now):
                 see_device(mac, result, rssi)
         except bluetooth.BluetoothError:
             _LOGGER.exception("Error looking up Bluetooth device")
-        track_point_in_utc_time(
-            hass, update_bluetooth, dt_util.utcnow() + interval)
+        if not once:
```

Comment:
I would suggest extract line 85-102 to a new function. And call that function from `update_bluetooth` and `handle_update_bluetooth`. The you can avoid the `once` parameter

Reply:
Sounds good, addressed with 5b4bfa4